pandas/core/interchange/dataframe.py

```diff
@@ -82,7 +87,7 @@ def select_columns(self, indices) -> PandasDataFrameXchg:
             self._df.iloc[:, indices], self._nan_as_null, self._allow_copy
         )
 
-    def select_columns_by_name(self, names) -> PandasDataFrameXchg:
+    def select_columns_by_name(self, names: Sequence[str]) -> PandasDataFrameXchg:
```

Comment:
This would allow `str`  @Dr-Irv , does it work for a string at runtime?

Reply:
I don't think so. I think it's intended to accept a list-like of strings, but I just matched the base class here. Thoughts?

Reply:
If you only want to accept a list of strings, you have to use `list[str]`.  If you use `Sequence[str]` then an argument of `"abc"` would match